apps/test/integration/dropletIntegrationTests.js

```diff
@@ -0,0 +1,61 @@
+import {assert, expect} from '../util/configuredChai';
+import * as dropletUtils from '@cdo/apps/dropletUtils';
+import {singleton as studioApp, stubStudioApp, restoreStudioApp} from '@cdo/apps/StudioApp';
+import {loadSource} from './levelTests';
+
+describe('setParamAtIndex', () => {
+  let editor, parser, plainTree, arrayTree;
+
+  before(function (done) {
+    // Load droplet sources.
+    loadSource('/base/lib/ace/src-noconflict/ace.js')
+    .then(function () { return loadSource('/base/lib/ace/src-noconflict/mode-javascript.js'); })
+    .then(function () { return loadSource('/base/lib/ace/src-noconflict/ext-language_tools.js'); })
+    .then(function () { return loadSource('/base/lib/droplet/droplet-full.js'); })
+    .then(function () { return loadSource('/base/lib/tooltipster/jquery.tooltipster.js'); })
+    .then(function () { return loadSource('/base/lib/phaser/phaser.js'); })
+    .then(function () {
+      assert(window.droplet, 'droplet in global namespace');
+      done();
```

Comment:
A [`mocha`](https://mochajs.org/) convention you may find useful here: You can [return a Promise instead of using the `done()` callback](https://mochajs.org/#working-with-promises).

```